torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -460,6 +476,9 @@ def generate_detector_report(self, model: GraphModule) -> Tuple[str, Dict[str, A
 
         modules_added: bool = False  # check to make sure at least 1 module added.
 
+        dynamic_benefit = " You will get more accurate results if you use dynamic quantization"
+        static_benefit = " You can increase model efficiency if you use static quantization"
+        future_support_str = ". This layer is not yet supported for dynamic quantization but will be soon"
```

Comment:
nit: maybe remove `will be soon`, we are not sure when we'll support dynamic conv actually